index.html

```diff
@@ -7,7 +7,17 @@
 </head>
 
 <body>
-
+    <h1>The Black Binder</h1>
+    <h2>Your number 1 directory for Black Culture</h2>
+    <br/>
+    <p>Enter your email address and we will let you know when we are almost ready to go live</p>
+    <form action="/action_page.php">
+        Email: <input type="text" name="Email" value="">
```

Comment:
Why is **Email:** outside of the `<input>`?

Can you write something in the `value` for the placeholder?